dataprofiler/tests/labelers/test_integration_column_name_data_labeler.py

```diff
@@ -0,0 +1,69 @@
+import os
```

Comment:
brand new file for testing the pre processor, model, and post processor all together